### ISSUE TITLE
Remember scroll position in preview mode

### ISSUE THREAD

**Issue by veesar** [stefan-niedermann/nextcloud-notes#1046]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Describe the bug**

The app doesn't remember the scroll position when viewing a note in Preview mode. This feature was working in previous versions and seems to be an issue with the current version. Scroll position is remembered when in Edit mode.

**To Reproduce**
(App Settings: Behavior > Display mode for notes: Open in preview mode)
1. Click on a note from the list
2. Opens in preview mode
3. Scroll down to position
4. Tap back
5. Open the note again
6. Scroll position is at the top of the note
7. Tap the Pencil icon to edit
8. Scroll moves to the last viewed position
9. Tap the Eye (Preview) icon
10. View moves to the top of note

**Expected behavior**

When opening a note, note should scroll to last viewed position

**Screenshots**

**Smartphone (please complete the following information):**
 - Nextcloud Notes-Version (android app): 3.1.0
 - F-Droid or Play Store: F-Droid
 - Android-Version: 10
 - Device: Redmi K20 Pro

**Server**
 - Nextcloud version: 20.0.5
 - Nextcloud Notes version (server app): 3.6.4

**Comment by stefan-niedermann** [comment]
Good catch, i'll have a look 🙂

**Comment by stefan-niedermann** [comment]
First analysis shows, that apparently not the `ScrollView` is scrolling anymore since the update, but the `TextView` of the markdown editor itself.
Using

```java
binding.singleNoteContent.post(() -> binding.singleNoteContent.scrollTo(0, scrollY));
```
instead of

```java
scrollView.post(() -> scrollView.scrollTo(0, scrollY));
```

works.

However this needs more investigation because it makes me also wonder 
- how to get the value which is actually *stored* as `ScrollY` value - maybe we need to fetch that one too from the `TextView`
- whether we need different implementations for the `NoteEditFragment` and the `NotePreviewFragment` instead of the generic one in `BaseNoteFragment`

**Comment by stefan-niedermann** [comment]
Will be fixed in `v3.2.5`.

**Comment by MiMoHo** [comment]
The issue was re-introduced presumably in v.4.4.0. When leaving a note and re-entering it, the last scrolling position is not remembered but the note opens at the top. Could you fix it again @stefan-niedermann 🙏
